src/components/Test.js

```diff
@@ -1,183 +1,183 @@
+        <div class="modal-dialog">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h1 class="modal-title fs-5" id="signupModal">Add Your Todo</h1>
+              <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <div class="modal-body">
+              <form>
+                <label>Enter Your Todo</label>
+                <TextArea type="text" value={newTodo} onChange={(e) => setNewTodo(e.target.value)}></TextArea>
+                <label >Status</label>
+                <select key={"index"} className="my-3 mx-4" onChange={FilterStatus}>
+                  <option value="complete">Complete</option>
+                  <option value="incomplete">Incomplete</option>
+                </select>
+                <div class="modal-footer">
+                  <Button className="mx-4" style={{ float: 'left' }} type="primary" onClick={handleAddTodo}>Add Todo</Button>
+                </div>
+              </form>
+            </div>
+          </div>
+        </div>
+      </div>
+
+
+
+      <div className="container my-3" style={{ border: '.2rem solid gray', backgroundColor: '#cccccc' }}>
+        <h2 className="text-center">Your ToDo</h2>
+        {/* Display when todos lenght is less then 1 */}
+         {todos.length < 1 ? (
+          <p className="text-center " >Add your todo</p>
+        ) : null}
+        <div className="container" >
+          <ul style={{ listStyle: 'none'}}>
+            {/* Filter todos on the bases of the options */}
+          {todos.filter((todo) => {
+  if (filter === "complete") {
+    return todo.status === "complete";
+  } else if (filter === "incomplete") {
+    return todo.status === "incomplete";
+  } else {
+    return true; // Filter is set to "All", show all todos
+  }
+}).map((todo, index) => (
+  <li key={index} className="my-4" style={{backgroundColor:'white',padding:'2rem'}}>
+    {/* Updateing todo if condition true */}
+   {editIndex === index ? (
+    <Form>
+      <Form.Item>
+        <TextArea type="text" value={editTodo} onChange={(e) => setEditTodo(e.target.value)}/>
+      </Form.Item>
+      <Form.Item>
+      <label>Status</label>
+      <select value={editStatus} className="my-3 mx-4" onChange={(e) => setEditStatus(e.target.value)}>
+       <option value="complete">Complete</option>
+      <option value="incomplete">Incomplete</option> 
+    
+    </select>
+      </Form.Item>
+</Form>
+   ) : (
+   <span className="d-flex">
+    {/* check box to show which todo is complete or not */}
+    <Checkbox checked={todo.status === 'incomplete' ? false : true}></Checkbox>
+  <p style={{ fontWeight: 'bold' }} className="mx-3">Title:</p>
+  <p className="mx-3" style={{ textDecoration: todo.status === 'incomplete' ? '' : 'line-through' }}>
+    {todo.title}</p>
+</span>)}
+<p>Updated On: {SetDate()}</p>

```

Comment:
why are you using the current time `now()` ? as update will get the current time whenever you load/refresh page